docs/index.rst

```diff
@@ -1,26 +1,15 @@
-
-.. mdinclude:: readme.md
-
-Contents
-========
+Welcome to MMCV's documentation!
+================================
 
 .. toctree::
    :maxdepth: 2
 
-   io.md
-   image.md
-   video.md
-   visualization.md
-   utils.md
-   runner.md
-   registry.md
-   cnn.md
-   ops.md
-   build.md
+   getting-started.rst
    deployment.rst
-   trouble_shooting.md
+   understand-mmcv.rst
    api.rst
-
+   trouble_shooting.md
```

Comment:
trouble_shooting -> FAQ, also change the title from trouble shooting to Frequently Asked Questions.